native-server/src/main/java/com/datastax/oss/simulacron/server/BoundNode.java

```diff
@@ -412,6 +412,7 @@ void handle(ChannelHandlerContext ctx, Frame frame) {
       } else if (frame.message instanceof Query) {
         Query query = (Query) frame.message;
         String queryStr = query.query;
+        logger.warn("No stub mapping found for message type QUERY: \"{}\"", queryStr);
```

Comment:
We might want to avoid logging this message when a `USE _KEYSPACE_` request is executed. Note that right after this line there is some logic to handle these requests.

The driver typically sends those out when a user specifies the default keyspace so it could result in a lot of log messages that are not useful.

Reply:
good point, will move it down to only log when the query is not a 'USE' query